UPGRADE.md

```diff
@@ -2,6 +2,16 @@
 
 ## 2.3
 
+### Change entityId field in AccessControl entity to string
```

Comment:
think we need to add a section for executing the `sulu:security:sync-phpcr-permissions` command

Reply:
fixed